ui/decredmaterial/collapsible.go

```diff
@@ -111,13 +111,20 @@ func (c *Collapsible) IsExpanded() bool {
 	return c.isExpanded
 }
 
-func (c *CollapsibleWithOption) Layout(gtx layout.Context, header, body func(C) D, more func(C)) layout.Dimensions {
+var rememberExpand map[int]bool
+
+func (c *CollapsibleWithOption) Layout(gtx layout.Context, header, body func(C) D, more func(C), walletID int) layout.Dimensions {
+	if rememberExpand == nil {
+		rememberExpand = make(map[int]bool)
+	}
```

Comment:
seeing as this isn't used outside this method, we can declare and initialize here
```suggestion
	rememberExpand := make(map[int]bool)
```

Reply:
or does it override the initial saved map? each time the layout is drawn?